test/test_distributions.py

```diff
@@ -1,10 +1,13 @@
-from common import TestCase, run_tests
 import math
-import torch
 import unittest
+import warnings
 from itertools import product
+
+import torch
+from common import TestCase, run_tests
 from torch.autograd import Variable, gradcheck
-from torch.distributions import Bernoulli, Categorical, Normal, Gamma, Distribution
+from torch.distributions import (Bernoulli, Beta, Categorical, Dirichlet,
```

Comment:
I'll switch to that if you prefer.

Reply:
Egads, that leads to lots of `pyflakes` warnings. Would you mind if we keep the exhaustive list?

Reply:
We never use `import *` in Pyro and I avoid it in my own code since it's so dangerous. E.g. it's unclear whether a symbol from `import *` shadows a previous import.

Reply:
That is true, although we're using them in a few places and didn't have problems so far. It was just a suggestion, I'm ok with listing them explicitly

Reply:
Ok thanks.